File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PackagesPackageSummaryProductLabel.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PackagesPackageSummaryProductLabel {
    @JsonProperty("IsActive")
    public Boolean isActive;
    @JsonProperty("IsChildFromParentWithLabel")
    public Boolean isChildFromParentWithLabel;
    @JsonProperty("IsDiscontinued")
    public Boolean isDiscontinued;
    @JsonProperty("LastLabelGenerationDate")
    public String lastLabelGenerationDate;
    @JsonProperty("PackageId")
    public Integer packageId;
    @JsonProperty("QrCount")
    public Integer qrCount;
    @JsonProperty("TagId")
    public Integer tagId;
}

File: sdks/thunkmetrc-rs/wrapper/src/models/packages_package_summary_product_label.rs
use serde::{Deserialize, Serialize};
use super::*;

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct PackagesPackageSummaryProductLabel {
    #[serde(rename = "IsActive", skip_serializing_if = "Option::is_none")]
    pub is_active: Option<bool>,
    #[serde(rename = "IsChildFromParentWithLabel", skip_serializing_if = "Option::is_none")]
    pub is_child_from_parent_with_label: Option<bool>,
    #[serde(rename = "IsDiscontinued", skip_serializing_if = "Option::is_none")]
    pub is_discontinued: Option<bool>,
    #[serde(rename = "LastLabelGenerationDate", skip_serializing_if = "Option::is_none")]
    pub last_label_generation_date: Option<String>,
    #[serde(rename = "PackageId", skip_serializing_if = "Option::is_none")]
    pub package_id: Option<i64>,
    #[serde(rename = "QrCount", skip_serializing_if = "Option::is_none")]
    pub qr_count: Option<i64>,
    #[serde(rename = "TagId", skip_serializing_if = "Option::is_none")]
    pub tag_id: Option<i64>,
}
